packages/cli/src/commands/push.ts

```diff
@@ -82,6 +86,18 @@ export async function handlePush(argv: PushArgs): Promise<void> {
     );
   }
 
+  if (batchId && !batchId.replace(/\s/g, '')) {
```

Comment:
Maybe it can be simplified to !batachId.trim() ?

Reply:
Make sense. Added.